textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/join/Join.java

```diff
@@ -70,9 +70,12 @@ public Join(IOperator outerOperator, IOperator innerOperator, JoinPredicate join
 
 	@Override
 	public void open() throws Exception, DataFlowException {
-		if (!(joinPredicate.getjoinAttribute().getFieldType().equals(FieldType.STRING)
-				|| joinPredicate.getjoinAttribute().getFieldType().equals(FieldType.TEXT))) {
-			throw new Exception("Fields other than \"STRING\" and \"TEXT\" are not supported by Join yet.");
+		if (!(joinPredicate.getjoinAttribute().getFieldType()
```

Comment:
I think Sripad did some re-formatting to make the code look better in Eclipse.

As the professor says, try not to make these changes when the logic is changed. We can write logic in a PR, and cleanup the code, do some formatting in another PR.